### ISSUE TITLE
Setting cordova-plugin-* params within config

### ISSUE THREAD

**Issue by Aaronfischer** [isleofcode/corber#443]
Perhaps this is a ridiculous question, but I'm not sure how to set the cordova-plugin-* params within the config file...

Example, cordova-plugin-geolocation within `/corber/cordova/config.xml`:
```js
<plugin name="cordova-plugin-geolocation" spec="~2.4.2">
    <variable name="GEOLOCATION_USAGE_DESCRIPTION" value=" " />
</plugin>
```
When I go and set `value` and then run `corber build --platform=ios` it erases the `value` that was set, back to the empty string you see above.

**Comment by alexblom** [comment]
I am unable to replicate this - will reach out to you on slack to see the issue. 

I'm able to add the config tag in config.xml - and after running a corber build it is not blanked.

**Comment by Aaronfischer** [comment]
Hey @alexblom responded to you on slack but for "archival" purposes I fixed things with these steps:
1. Uninstalled the cordova-plugin (ensuring that the actual cordova plugin folder was also removed from /corber/cordova/plugins/PLUGIN
2. Reinstalled via `corber plugin add cordova-plugin-geolocation --save`
3. Added the tags (the syntax adding the geolocation_usage changed slightly with a plugin upgrade)
4. On `corber build --platform=ios`
5. All is good.

So my assumption since it worked fine for you the first time is that the uninstall and reinstall is what did the trick in my situation.

Thanks!
